week3/final/robot-text/robot.js

```diff
@@ -62,11 +69,15 @@
 
     const cellContents = board[y][x];
 
-    if (cellContents === '.' || cellContents === 'F') {
+    if (cellContents === '.' || cellContents === 'F' || cellContents==='A') {
```

Comment:
Add a space before and after the `===`. This happens automatically if you use the Format Document command in VSCode.